docs/documentation/siddhi-4.0.md

```diff
@@ -46,6 +46,208 @@ by a semicolon `( ; )` as shown in the below syntax.
         ;
 ```
 
+**Performance**
+
+To evaluate the performance of an application, you can enable the statistics of a Siddhi application to be published. This is done via the `@app` annotation that can be added to a Siddhi application as shown in the following example.
+
+```sql
+@app:statistics(reporter = 'console')
+```
+The following elements are configured with this annotation.
+
+|Annotation| Description| Default Value|
+| ------------- |-------------|-------------|
+|`reporter`|The type of the reporter for statistics. Possible values are as follows:<br/> `console`<br/> `jmx`<br/> `jdbc`<br/> If `jdbc` is specoified, the statistics are stored in the `WSO2_METRICS_DB` datasource (which is configured in the `<SP_HOME>/conf/<PROFILE>/deployment.yaml` file. |`console`|
```

Comment:
Let's add possible values as only console and jmx. jdbc is only available in Stream processor.